README.md

```diff
@@ -1,11 +1,314 @@
+
+If everything is correct, MANA equivalent to the pricePerDay index I've selected, times the rentalDays I provided will be transfered from my address to the lessor (and a fee to the DAO) and the provided operator will start being able to deploy scenes to that LAND.
+
+## Offers
+
+This is the contrary of a [Listing](#listing). In this case, me, as a user interested of renting a certain LAND, would like to sign an Offer for said LAND so the owner has the possibility of renting it to me.
+
+Similar to the Listing, I would need to sign the Offer conditions. These conditions can be seen in the Offer struct of the Rentals contract:
+
+<a id="offer-struct"></a>
+```
+struct Offer {
+    address signer;
+    address contractAddress;
+    uint256 tokenId;
+    uint256 expiration;
+    uint256[3] nonces;
+    uint256 pricePerDay;
+    uint256 rentalDays;
+    address operator;
+    bytes32 fingerprint;
+    bytes signature;
+}
+```
+
+- signer: The address of the signer, in this case, my address.
+- contractAddress: The address of the LAND contract.
+- tokenId: The token id of the LAND I want to rent.
+- expiration: The timestamp from which the signature will not be valid anymore.
+- nonces: The three of them have to match the current nonces in the Rentals contract to be valid. Find more about it in the [Nonces](#nonces) section.
+- pricePerDay: Defines the price per day I'm willing to pay for renting the LAND
+- rentalDays: The amount of days I want to rent the LAND
+- operator: The address that will be given operator permissions to deploy scenes to that LAND
+- fingerprint: If the asset would be an Estate, the fingerprint of that Estate should match the one I'm providing so the owner cannot remove LAND from it before renting it to me.
+- signature: The signature created by signing all the previous data
+
+There are various ways of signing these conditions. One of them can be achieved by using ethers as seen in the [./test/utlils/rentals.ts](https://github.com/decentraland/rentals-contract/blob/main/test/utils/rentals.ts#L81) utility file.
+
+Once both the signature and the listings conditions are ready, I, or whichever off-chain handling this, can store them to make them 
+available for the interested owner of that LAND to accept it.
+
+If the owner of the the LAND is interested in the offer, they can interact with the Rentals contract via `acceptOffer` or by sending the asset to the Rentals contract via the `safeTransferFrom` function in the LAND contract, providing the Offer in the last `bytes` parameter. More info on this can be found in the [Accepting an Offer](#accepting-an-offer) section.
+
+## Accepting an Offer
+
+Me, as the current owner of a LAND, see that there is an offer by a much generous user that wants to rent my LAND for a year and for a lot of MANA.
```

Comment:
This is a bit weird as `Me` does not function the same way as `I` in this context. The clarification between the comas make it seem as if the verb should be in third person (sees) but the subject (me) is in first. Thing is that `Me see` is not correct english.

maybe 
```suggestion
When I, as the current owner of a LAND, see that there is an offer by a much generous user that wants to rent my LAND for a year and for a lot of MANA.
```